src/pages/documentation/release-notes/index.md

```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## October 11, 2022
+
+### iOS AEPPlaces 3.0.2
+
+* Non-string values in Point of Interest metadata are now correctly handled.
```

Comment:
looks like there is no release-notes file under https://developer-stage.adobe.com/client-sdks/documentation/places/ as for the other extensions.
@sbenedicadb do we usually update this space with release notes https://experienceleague.adobe.com/docs/places/using/release-notes.html ?

Reply:
never had that page in sdkdocs - i never understood why we were duplicating that information to begin with 😬

Reply:
I know some Launch cards reference the release notes link under the extension space and it helps with filtering out the content for that particular extension, in that sense could be useful for Places too. 
I also agree might be a good time to discuss if the main release-notes page is still needed in that case, or how to handle it without duplication. Do you have any suggestions or want to take this offline?